spec/lib/saml/ssoe_user_spec.rb

```diff
@@ -758,7 +768,8 @@
           birls_id: '796123607',
           icn: '1012740600V714187',
           multifactor: multifactor,
-          common_name: 'dslogon10923109@gmail.com'
+          common_name: 'dslogon10923109@gmail.com',
+          person_types: ['PAT']
```

Comment:
Do we want a spec for multiple person types?

Reply:
Oh yeah, might be worth adding a spec for multiple person types to test the parsing logic

Reply:
added multiple person types to the same test and checked to ensure that it comes back as an array of the names.